ember-phone-input/src/components/phone-input.ts

```diff
@@ -29,10 +43,70 @@ import '../styles/styles.css';
   @public
 */
 
-export default class PhoneInputComponent extends Component {
-  @service phoneInput;
+export interface PhoneInputArgs {
+  country: string;
+  number: string | null;
```

Comment:
Would `number?: string` work here?

Reply:
That would change the type of number to string | undefined which contradicts the goal of this MR. I think this MR should migrate types as they were without any changes (especially args types), and maybe improve them in another upcoming MR. Wdyt?